templates/docs/patterns/chip.md

```diff
@@ -14,6 +14,23 @@ Our chip component is to be used to display small actionable pieces of informati
 View example of the default chip pattern
 </a></div>
 
+Chips come in 5 colours. The default is neutral (grey). Use any of the following flavours to style valuesthat have semantic or otherwise colour coded meaning:
```

Comment:
typo: 

```suggestion
Chips come in 5 colours. The default is neutral (grey). Use any of the following flavours to style values that have semantic or otherwise colour coded meaning:
```

Reply:
fixed